### DISCUSSION TITLE
Cannot reapply for student education pack

### DISCUSSION THREAD

**Discussion by HarryAllen1** [community/community#127851] [GitHub Education] [answered]
### Hi everyone,

A few years ago, I applied and got accepted for the Student Developer pack. However, on 20 June, this will expire, so I followed the link provided to try and reapply. I entered my school and got sent to the page where I was asked to upload my student ID. I tried to do this, but when I clicked the "Process my application" button, the page refreshes and nothing happens. What should I try to do to fix this?

**Comment by Sodiumchloridy** [comment]
I used to have this problem once too, for me it's the billing information, they started to require billing information (it does NOT require payment method) on my 2nd application. Do so and refresh then try again, good luck.

> Complete your GitHub account [billing information](https://github.com/settings/billing/payment_information) with your full legal name as it appears on your academic affiliation documentation. (You do not have to add a payment method.)

As stated in https://education.github.com/discount_requests/application:
> Before you begin:
> - Check that you are using a [supported browser](https://docs.github.com/en/get-started/using-github/supported-browsers), and that location services are not blocked by your browser or platform.
> - Complete your GitHub account [billing information](https://github.com/settings/billing/payment_information) with your full legal name as it appears on your academic affiliation documentation. (You do not have to add a payment method.)
> - [Verify](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/adding-an-email-address-to-your-github-account) your academic email address on your GitHub account, if your school provides one.
> - Secure your GitHub account with [two-factor authentication](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa). (We recommend using the [GitHub Mobile](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication#configuring-two-factor-authentication-using-github-mobile) app.)
> - [Personalize](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) your public GitHub Profile with your photo, your name, your pronouns, and more.
> - Generate a README for your public GitHub Profile, [customize it with your interests](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme), and publish it for the world to see.

Hope it helps, good luck.

**Reply by coregano** [reply]
I have the same problems as OP and updating the billing information does not work. I have also signed in and out to retry but the same issue persists. 

Does anyone else have any other solutions?

**Reply by ItzDc02** [reply]
Happened to me too. I have put my info on the billing thing but I still got instantly rejected and they didnt even gave a reason this time around

**Reply by HarryAllen1** [reply]
I also tried adding my billing info (and a payment method) and still cannot re-apply.

**Reply by ItzDc02** [reply]
Idk why the posts have been unanswered by the mods for so long even though a lot of people are posting about the same issue almost everyday

**Comment by HarryAllen1** [comment, marked as answer]
For me, what worked was taking the picture of my student ID directly through the application site.

**Comment by CellMelton** [comment]
![image](https://github.com/user-attachments/assets/b89c8840-cdca-4c2b-991d-b3e3c9fab3c1)
it won't allow me to upload a file of my transcripts... help :(

**Comment by CHIVY2005** [comment]
i still can not reapply, i have done all the stuff such as refilled my billing to correct, anble 2fa both mobile and 2fa app, and then change information on my personal page to correct syntax but whenever i log out and sign back it still the denied statues and i sill can not reapply please help me

**Reply by CHIVY2005** [reply]
please help me, i have apptempted many times but it can not be processed. i'm a real student and i really and to have this pack. All my friends had had it but i still can not have it. I also can not send ticket and copilot guided all the stuffs i have tried. I am so tired just because it
